packages/block-editor/src/components/block-list/style.scss

```diff
@@ -104,8 +100,29 @@
 			border-color: var(--wp-admin-theme-color);
 		}
 	}
-}
 
+	// Ensure an accurate partial text selection.
+	// To do this, we disable text selection on the main container, then re-enable it only on the
+	// elements that actually get selected.
+	// To keep in mind: user-select is currently inherited to all nodes inside.
+	user-select: none;
+
+	// Re-enable it on components inside.
+	[class^="components-"] {
+		user-select: text;
+	}
+
+	// Re-enable it on editable items and a few others.
+	.block-editor-block-list__block,
+	[contenteditable] {
+		user-select: text;
```

Comment:
What is the point of adding `[contenteditable]` if it's already enabled on the block.
That actually seems like a god solution: disable text selection for block lists and re-enable it for blocks.

Reply:
Good call, I removed contenteditable and just targetting blocks. I had to move the code a bit.

Reply:
Which components are in a list of blocks that is not inside a block?

Reply:
The components reference is meant to target content inside placeholders.

Reply:
Aren't these inside `.block-editor-block-list__block`? I don't get this extra rule, sorry.

Reply:
@jasmussen I'm still not sure what this rule is needed btw.

Reply:
You're referring to `[class^="components-"]`? It's needed to enable text selection inside placeholders like Cover. It's not that you need to select text there that often, but right now the multi-select still selects that text, and without a visible selection marker, it becomes confusing why the multi-select isn't invoking. Possibly related to this: https://github.com/WordPress/gutenberg/pull/42983#issuecomment-1228441403